README.md

```diff
@@ -37,15 +37,23 @@ Last but not least, we're thankful to these companies for offering their service
 ### Shortcuts
 
 You can almost do anything with shortcuts. Click on the help icon on the bottom right corner to see them all.
+<img height="500" width="500" src="./public/screenshots/shortcuts_help.png" alt="shortcut demo for help button" /><img height="500" width="500" src="./public/screenshots/shortcuts.PNG" alt="shortcut demo for shortcuts menu" />
```

Comment:
Better to place images next to each other so it looks better visually. Also can you highlight only the "?" icon and make the outline uniform as well ?
![shortcuts](https://user-images.githubusercontent.com/11256141/144971082-9043c3b5-89c2-4be6-924c-94ebdadab157.png)